src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,626 @@
+import _ from 'lodash';
+import React from 'react';
+import fullSchema36 from 'vets-json-schema/dist/28-8832-schema.json';
+
+import dateUI from '../../../common/schemaform/definitions/date';
+import * as address from '../../../common/schemaform/definitions/address';
+import SSNWidget from '../../../common/schemaform/widgets/SSNWidget';
+import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
+import fullNameUI from '../../../common/schemaform/definitions/fullName';
+import ssnUI from '../../../common/schemaform/definitions/ssn';
+import { genderLabels } from '../../../common/utils/labels';
+
+import VerifiedReviewContainer from '../components/VerifiedReviewContainer';
+import {
+  PrimaryAddressViewField,
+  getPage,
+  VAFileNumberDescription
+} from '../helpers';
+
+import initialData from '../../../../../test/disability-benefits/526EZ/schema/initialData';
+
+function isValidZIP(value) {
+  if (value !== null) {
+    return /^\d{9}$/.test(value) || /^\d{5}-\d{4}$/.test(value);
+  }
+  return true;
+}
+
+function isValidPhone(value) {
+  if (value !== null) {
+    return /^\d{10}$/.test(value) || /^\d{11}$/.test(value);
+  }
+  return true;
+}
+
+function validatePhone(errors, phone) {
+  if (phone && !isValidPhone(phone)) {
+    errors.addError(
+      'Phone numbers must be at least 10 digits (dashes allowed)'
+    );
+  }
+}
+
+function validateZIP(errors, zip) {
+  if (zip && !isValidZIP(zip)) {
+    errors.addError('Please enter a valid 9 digit ZIP (dashes allowed)');
```

Comment:
@peggygannon Could you please weigh in on a proposed update to this validation message (as the second portion of the ZIP is not required it turns out):  `Please enter a valid 5 or 9 digit ZIP (dashes allowed)`
Also, you previously proposed replacing ZIP with postal code, as this field is only collected for US addresses, should we keep it as ZIP or replace it with postal code?

Reply:
@goldenmeanie recommends `Please enter a valid ZIP code`